src/inputgroup/skin.css

```diff
@@ -86,3 +102,77 @@
     }
   }
 }
+
+.spectrum-Datepicker--range {
+  .spectrum-InputGroup-field {
+    &:focus-ring {
+      box-shadow: none !important;
+    }
+    &[disabled] {
+      ~ hr {
+        color: var(--spectrum-textfield-text-color-disabled);
+      }
+    }
+  }
+
+  /* Focus ring */
+  :focus-ring {
```

Comment:
This might also need to be `&:focus-ring`?

Reply:
As above In this case, we're creating a selector for `.spectrum-Datepicker-focusRing` following and adjacent to any keyboard focused sibling. `.spectrum-Datepicker--range :focus-ring ~ .spectrum-Datepicker-focusRing`. Either of the `input` elements or the `button` can have `:focus-ring`.